lib/network.sh

```diff
@@ -74,8 +74,8 @@ export CLUSTER_BARE_METAL_PROVISIONER_IP
 
 # shellcheck disable=SC2153
 if [[ "$BARE_METAL_PROVISIONER_IP" = *":"* ]]; then
-  export BARE_METAL_RPOVISIONER_URL_HOST="[$BARE_METAL_PROVISIONER_IP]"
-  export CLUSTER_BARE_METAL_PROVISIONER_URL_HOST="[$CLUSTER_BARE_METAL_PROVISIONER_IP]"
+  export BARE_METAL_PROVISIONER_URL_HOST="[$BARE_METAL_PROVISIONER_IP]"
```

Comment:
```suggestion
  export BARE_METAL_PROVISIONER_URL_HOST="[${BARE_METAL_PROVISIONER_IP}]"
```
optional: While touching the line, might fix it as well.